pep/app/services/current-user.ts

```diff
@@ -413,4 +414,21 @@ export default class CurrentUserService extends Service {
     updateTextJustification(textJustification: TextJustificationId) {
         return this.updatePrefs({ [PreferenceKey.TEXT_JUSTIFICATION]: textJustification });
     }
+
+    modifyWidgetConfigurations(widgetConfigurations: WidgetConfiguration[]) {
+        const configs = this.preferences?.widgetConfigurations;
+        let updatedConfigs = configs;
+        widgetConfigurations.forEach((configuration) => {
+            const configAlreadyExists = configs?.find((item) => item.widget === configuration.widget) ?? false;
```

Comment:
```suggestion
            const configAlreadyExists = configs?.some((item) => item.widget === configuration.widget);
```
`array.some()` returns a boolean, which it looks like is a better match for what you're doing here.